tests/integration/test_kafka_bad_messages/test.py

```diff
@@ -48,8 +94,9 @@ def test_bad_messages_parsing_stream(kafka_cluster):
         "MySQLDump",
     ]:
         print(format_name)
+        topic_name = f"{format_name}_{handle_error_mode}_err_{int(time.time())}"
```

Comment:
Have you thought about just truncating the system table instead of using the timestamp as postfix? For me that feels more natural, but I don't have a strong preference, so feel free to keep it as is.